File: src/main/java/dev/fromnowon/mfiserver/dto/TokenMapper.java
package dev.fromnowon.mfiserver.dto;

import org.mapstruct.Mapper;
import org.mapstruct.MappingConstants;
import org.mapstruct.ReportingPolicy;

@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE, componentModel = MappingConstants.ComponentModel.SPRING)
public interface TokenMapper {

    TokenDataDTO1 testConvert(TokenDataDTO tokenDataDTO);

}

File: src/main/kotlin/dev/fromnowon/mfiserver/dto/TokenMapper.kt
package dev.fromnowon.mfiserver.dto

import org.mapstruct.Mapper
import org.mapstruct.MappingConstants
import org.mapstruct.ReportingPolicy

@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE, componentModel = MappingConstants.ComponentModel.SPRING)
interface TokenMapper {

    fun testConvert(tokenDataDTO: TokenDataDTO?): TokenDataDTO1?

}